package.json

```diff
@@ -215,7 +212,7 @@
     "lint:css": "stylelint \"client/**/*.scss\" --syntax scss",
     "lint:js": "npm run -s install-if-deps-outdated && eslint-eslines .",
     "lint:mixedindent": "mixedindentlint --ignore-comments \"client/**/*.scss\" \"assets/**/*.scss\" \"**/*.js\" \"**/*.jsx\" \"!build/**\" \"!node_modules/**\" \"!public/**\" \"!client/config/index.js\"",
-    "minify": "./bin/minify",
+    "minify": "node bin/minify.js",
```

Comment:
is there any impact to this change?

Reply:
Only that now it works on Windows. Windows doesn't interpret the `#! /usr/bin/node` header in a file, so you have to be explicit and invoke it with `node ./path/to/file`.